src/style.css

```
@@ -0,0 +1,44 @@
+body {
+  margin: 0;
+}
+
+:root {
+  --star-active: url("./images/star-active.svg") center no-repeat;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars div:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url("./images/star.svg") center no-repeat;
+}
+
+.stars--1 div:first-child {
+  background: var(--star-active);
+}
+
+.stars--2 div:nth-child(1),
+.stars--2 div:nth-child(2) {
```

```suggestion
.stars--2 :nth-child(-n+2) {
```

Figure out with [nth-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child) rule

Fix everywhere

---

I did this in second commit lower.